packages/devextreme/js/__internal/ui/list/list.edit.strategy.grouped.ts

```diff
@@ -16,7 +16,7 @@ const SELECTION_SHIFT = 20;
 const SELECTION_MASK = (1 << SELECTION_SHIFT) - 1;
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-type GroupedItem = {
+export type GroupedItem = {
   [key: string]: unknown;
   key?: unknown;
   items?: unknown[];
```

Comment:
```suggestion
  items?: Item[];
```
also, look through code, perhaps it is possible to define type for key